### DISCUSSION TITLE
Onvif Client Test Tool Working

### DISCUSSION THREAD

**Discussion by CVVDN-S** [onvif/testspecs#18] [Q&A] [answered]
Hi,
Im new to this Onvif Client tool, i have earlier worked with Onvif Device Test Tool. But now i'm trying to check whether the NVR here is Profile S conformant or not. So i need to know how to run this Onvif Client Test Tool to test Profile S of NVR. Is there any Usermanual for this tool?
Any responses are welcomed, it will be great for me to start learning it.

**Comment by davidmneedham** [comment, marked as answer]
If you wish to check if a particular NVR meets Profile S, you may use the [Conformant Products Database](https://www.onvif.org/conformant-products/). For help in using the test tool itself, there is a built-in help function.
